process/block/helpers/txsorting.go

```diff
@@ -0,0 +1,15 @@
+package helpers
```

Comment:
this file could have been placed directly on process/block package so we could have avoid the `helper` package

Reply:
since this needs to be imported from different packages, it's better to be in a separate package to avoid circular dependencies